### DISCUSSION TITLE
SearchFor<Ticket> with query and users sideload

### DISCUSSION THREAD

**Discussion by spence524** [mozts2005/ZendeskApi_v2#577] [Q&A]
Appreciate any help as I am new to using this, but I am trying to return user information (ideally the requester's name) from this search, but I am not seeing it anywhere in the results.  Ultimately, what I am wanting to do is get a list of all unsolved tickets in a group and arrange them by Requester Name.  Here's a snippit of my code.

"query" is a parameter passed in to the function which is partly shown below.  query = "group:360003624953 status<solved";

tList = api.Search.SearchFor<Ticket>(query + "&include=tickets(users)", page: x);

**Comment by mozts2005** [comment]
At this time side loading with the search api is not supported by this library. I think support could be added and a Pull Request to add support would be welcomed.
